apps/docs/pages/guides/getting-started/tutorials/with-kotlin.mdx

```diff
@@ -268,16 +268,20 @@ class ProductRepositoryImpl @Inject constructor(
 
     override suspend fun getProduct(id: String): ProductDto {
         return withContext(Dispatchers.IO) {
-            postgrest.from("products").select {
-                eq("id", id)
+            postgrest["products"].select {
+                filter {
+                    eq("id", id)
+                }
             }.decodeSingle<ProductDto>()
         }
     }
 
     override suspend fun deleteProduct(id: String) {
         return withContext(Dispatchers.IO) {
-            postgrest.from("products").delete {
-                eq("id", id)
+            postgrest["products"].delete {
+                filter {
+                    eq("id", id)
+                }
```

Comment:
So the docs now only mention the `Supabase#from` / `Postgrest#from` method to make postgrest requests rather than `Postgrest#get` (the operator function). Maybe it would be less confusing if we are also using them here?

Reply:
That makes sense. Just updated